### ISSUE TITLE
Add taxonomy definition to documentation

### ISSUE THREAD

**Issue by rab497** [hughbris/grav-theme-landed#4]
It took me a few hours of delving into the twig templates and Grav documentation to understand that for the demo home page to show the sidebar sections, rather than simply ignoring them I need to add 'pagetype' to the defined taxonomy types in the site.yaml
Maybe update the documentation to mention this requirement in the installation notes.
May save someone a lot of time, and increase the uptake of this great looking theme.

**Comment by hughbris** [comment]
I agree, that's a good addition to make.

Plus I will look at supplying this setting in the demo and making sure it works from that. I believe [you can do this](https://learn.getgrav.org/16/advanced/grav-development#demo-content).

Thanks for the heads up :)

**Comment by rab497** [comment]
Excellent news. Thanks for being so speedy.

**Comment by hughbris** [comment]
Added cautionary note to latest develop branch now.

Strangely I had already included `pagetype` in [demo site.yaml](https://github.com/hughbris/grav-theme-landed/blob/a16c3e6f389a458af138c0188c1651ceecc770f0/_demo/config/site.yaml#L13) so I guess I will need to test and understand how that works better :/

Will leave this issue open until I've had a chance to explore that.

**Comment by rab497** [comment]
I installed using gpm and said yes to including the demo content. Don't know why that would not have worked!

**Comment by hughbris** [comment]
Yeah I'll set up some experiments to see what gpm does when there's existing headers that conflict :)
